docs/next.config.js

```diff
@@ -161,10 +160,7 @@ module.exports = withTypescript({
 
     return map;
   },
-  onDemandEntries: {
-    // Period (in ms) where the server will keep pages in the buffer
-    maxInactiveAge: 120 * 1e3, // default 25s
-    // Number of pages that should be kept simultaneously without being disposed
-    pagesBufferLength: 3, // default 2
+  async rewrites() {
+    return [{ source: '/api/:rest*', destination: '/api-docs/:rest' }];
```

Comment:
Should it be the opposite? The Netlify preview seems to use /api-docs instead of /api for the routes.

Reply:
Using as recommended: https://github.com/zeit/next.js/issues/9081#issuecomment-542280734

Seems to not work at all though. Reverse doesn't help, fixing `exportPathsMap` doesn't help.

Reply:
I'm probably misunderstanding something about that. It doesn't seem to be used at all. throwing an error doesn't even do something.

Reply:
Needs to be in an object called experimental

```js
experimental: {
  async rewrites() {
    return [{ source: '/api/:rest*', destination: '/api-docs/:rest' }];
  }
}
```

Reply:
Yeah the usual issues with nextjs and changelogs.